package.json

```diff
@@ -143,7 +145,7 @@
     "glob-parent": "^6.0.2",
     "netmask": "^2.0.1",
     "js-sha3": "^0.9.2",
-    "json-schema": "^0.4.0",
+    "jsonschema": "^1.4.1",
```

Comment:
Resolution makes sense 👍 Looks like it's pinned in a dependency for an unclear reason.

The removed `json-schema` seems unrelated though, an older unused resolution. I've removed it in a separate PR: https://github.com/MetaMask/metamask-extension/pull/30446